### PR TITLE
Add section about improving load times

### PR DESCRIPTION
**Pull request by rockdaboot**
Background: https://github.com/elastic/kibana/pull/142334

### CODE REVIEW

**Comment by iogbole** [comment]
@rockdaboot , let's chat about this on slack before we merge since this is a public repo. Got some questions.

File: README.md [hunk 1]
```diff
@@ -402,6 +402,14 @@ of the amount of logs produced.
 
 To enable debug logs, add the `-verbose` command-line flag or the `verbose true` setting in the configuration file.
 
+### Improve load times
+
+The amount of data to be loaded for the flamegraph, the topN functions and the traces view is often several megabytes.
```
Comment (mdbirnstiehl) [inline README.md:407]:
```suggestion
The amount of data loaded for the flamegraph, topN functions, and traces view can lead to latency when using a slow connection (e.g. DSL or mobile).
```

File: README.md [hunk 2]
```diff
@@ -402,6 +402,14 @@ of the amount of logs produced.
...
+This can lead to an annoying latency when acting on a slow connection (e.g. DSL or mobile).
```
Comment (mdbirnstiehl) [inline README.md:408]:
```suggestion

```

File: README.md [hunk 3]
```diff
@@ -402,6 +402,14 @@ of the amount of logs produced.
...
+
+If you already run Kibana/ES 8.6+, setting the Kibana cluster option `server.compression.brotli.enabled: true` reduces
+the amount of data to be transferred and thus it should reduce your time to wait.
```
Comment (mdbirnstiehl) [inline README.md:411]:
```suggestion
the amount of data to be transferred and should reduce load time.
```

**Review by mdbirnstiehl — APPROVED** [review]
Just a couple of wording suggestions.

**Comment by rockdaboot** [comment]
> Just a couple of wording suggestions.

Thank you, Mike !

**Review by iogbole — APPROVED** [review]
LGTM

**Pull request merged**